### DISCUSSION TITLE
Wie lassen sich einzelne Tabs in Addon-Seiten verstecken?

### DISCUSSION THREAD

**Discussion by eaCe** [FriendsOfREDAXO/.github#33] [Q&A] [answered]
Wie lassen sich einzelne Tabs (subpages), die über die package.yml definiert wurden, innerhalb der Addon-Seiten verstecken? 
Ein Tab soll ausgeblendet werden, wenn ein weiteres Addon installiert und aktiviert wurde.

**Comment by skerbis** [comment, marked as answer]
Seite entfernen z.B. in der boot.php des project AddOns

```php
$page = $this->getProperty('page');
unset($page['subpages']['subpage']);
$this->setProperty('page', $page);
```
Weitere Snippets dieser Art: https://friendsofredaxo.github.io/tricks/backend/backend_snippets
